vxdetector/tests/test_interact_bowtie.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/python
+
+import unittest
+import tempfile
+import os
+from glob import glob
+import vxdetector.interact_bowtie2 as ibo
+import shutil
+
+path = f'{os.path.dirname(__file__)}/'
+fasta_file = f'{path}test_data/5011_S225_L001_R1_001.fastq.gz'
+read2_file = f'{path}test_data/5011_S225_L001_R2_001.fastq.gz'
+test_unpaired = f'{path}test_data/unpaired/unpaired.sam'
+test_paired = f'{path}test_data/paired/paired.bed'
+
+samtools_path = shutil.which('samtools')
+if samtools_path is None:
+    samtools_path = '$CONDA/bin/samtools'
```

Comment:
global things are generally discouraged. Wouldn't it be sufficient to execute this in the `setUp` methods?